examples/algorithms/plot_girvan_newman.py

```diff
@@ -0,0 +1,74 @@
+"""
+=======================================
+Community Detection using Girvan-Newman
+=======================================
+
+This example shows the detection of communities in the Zachary Karate
+Club dataset using the Girvan-Newman method.
+
+We plot the change in modularity as important edges are removed. 
+Graph is coloured and plotted based on community detection when number 
+of iterations are 1 and 4 respectively.
+"""
+
+import networkx as nx
+from networkx.algorithms import community
```

Comment:
Just to make this more up-to-date with our "lazy-loading" system, can you remove this import and use `nx.community` everywhere later in place of `community`?

Reply:
Will do!